tests/subscriptions/test_worker.py

```diff
@@ -43,15 +46,8 @@ def all(self) -> Iterable[Tuple[UUID, SubscriptionData]]:
         return [*self.__subscriptions.items()]
 
 
-@pytest.fixture
-def dataset() -> Iterator[Dataset]:
-    with dataset_manager("events") as dataset:
-        yield dataset
```

Comment:
Why removing the fixture? It is a convenient way to reuse the logic that prepares a dataset with its storages and tear it down across tests.